decentraland/App.js

```diff
@@ -69,7 +69,8 @@ Is the config file correct?`;
     // On iOS environment, Secure Store data remains even after app uninstallation
     const isFirstUse = await retrieveIsFirstUse();
     if (isFirstUse) {
-      await storeAccount(null);
+      // TODO: Uncomment this when 'isFirstUse' flag is being set widely
+      //await storeAccount(null);
```

Comment:
Oh, rather than this I was picturing still removing the account if `isFirstUse`, but in the new app version I would set `isFirstUse` to `false` for all users on first app load and then remove that later

Reply:
I'll wait for your thoughts on this before merging, but I'm happy with leaving it as you've done it if you prefer

Reply:
Never mind, I'll merge this without waiting on this

Reply:
Hmmm, if that way works for you, it seems ok for me.
I think that sometime soon when we uncomment `await storeAccount(null);` we'll only remove accounts after uninstallation (the exception will be users that we'll pass all that timeframe without using the app).

Reply:
Yep, agreed. Sounds good. Resolving this.

Reply:
And noted about that caveat - I had thought about that but it’s okay.